nucypher/utilities/sandbox/blockchain.py

```diff
@@ -143,3 +155,24 @@ def time_travel(self, hours: int=None, seconds: int=None, periods: int=None):
         self.interface.w3.eth.web3.testing.timeTravel(timestamp=end_timestamp)
         self.interface.w3.eth.web3.testing.mine(1)
         self.log.info("Time traveled to {}".format(end_timestamp))
+
+    @classmethod
+    def connect(cls, *args, **kwargs) -> 'TesterBlockchain':
+        solidity_compiler = SolidityCompiler(test_contract_dir=CONTRACT_ROOT)
+        memory_registry = InMemoryEthereumContractRegistry()
+        interface = BlockchainDeployerInterface(provider_uri=cls._PROVIDER_URI, compiler=solidity_compiler, registry=memory_registry)
+        testerchain = TesterBlockchain(interface=interface, test_accounts=cls._default_test_accounts, airdrop=False)
+        return testerchain
+
+    @classmethod
+    def bootstrap_network(cls) -> Tuple['TesterBlockchain', List[str]]:
+
+        def __deploy_contracts(testerchain: TesterBlockchain) -> None:
```

Comment:
Why separate this logic into a function?  Why not just put these three lines in the place where this function is currently called?

Reply:
Honestly - I think this was re-purposed code - Yeah this doesn't need to be enclosed.

:heavy_check_mark: